core/script.runtime/src/kotlin/script/templates/standard/templates.kt

```diff
@@ -16,18 +16,25 @@
 
 package kotlin.script.templates.standard
 
+// discuss
+//
+// These are some 'basic' script templates
+// Should we keep them here?
```

Comment:
Since the (manual) construction of the script requires actual arguments array, the simple no-arg variant could be useful in some applications. So, id'leave it, may be renaming to `...NoParameters` (although I like `Simple` better).
The rest - I agree.